tests/e2e/configurer/chain/commands.go

```diff
@@ -216,44 +223,62 @@ func extractProposalIDFromResponse(response string) (int, error) {
 	return proposalID, nil
 }
 
-func (n *NodeConfig) BankSend(amount string, sendAddress string, receiveAddress string) {
-	n.BankSendWithWallet(amount, sendAddress, receiveAddress, "val")
+func (n *NodeConfig) BankSendError(amount string, sendAddress string, receiveAddress string, walletName string, gasLimit string, fees sdk.Coins, failError string) {
+	n.LogActionF("bank sending %s from address %s to %s", amount, sendAddress, receiveAddress)
+	cmd := []string{"terrad", "tx", "bank", "send", sendAddress, receiveAddress, amount, fmt.Sprintf("--from=%s", walletName)}
+	cmd = append(cmd, "--fees", fees.String(), "--gas", gasLimit)
```

Comment:
If you need to manually specify the `--fees` flag to make the test work, then the concept of tax2gas ist broken at this point - Because the point ist that you can get away with `--gas auto --gas-prices <price>` and the simulation/gas estimate will present to you the gas in such a way, that `gas estimate * gas prices >= tax + gas`

Reply:
Thanks, I will change them with gas-multiplier